### PR TITLE
feature/async redux setup

### PR DESCRIPTION
**Pull request by verywellced**
**Async Redux Setup**

- Generated AppState Freezed Files (Initial)
- Initialized Store (AppState)  on main function
- Made Homepage

HomePage

### CODE REVIEW

File: lib/home_page.dart [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+class HomePage extends StatelessWidget {
+  const HomePage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: Scaffold(
+        body: Center(
+          child: Text('This app is working'),
```
Comment (shawn-ffufm) [inline lib/home_page.dart:12]: maybe you can add a todo comment here as well? 🙂

File: notes.dart [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/*
+
+CHEAT CODE : Add the User Freezed form the old project 
+
+App State Setup
```
Comment (shawn-ffufm) [inline notes.dart]: @verywellced what's this all about? do u still need this code change here? 🙂

**Review by shawn-ffufm — CHANGES_REQUESTED** [review]
@verywellced see minor pr feedback 🙂

File: notes.dart (hunk 2 above)
Comment (verywellced) [reply notes.dart]: thats my notes sir, will transfer and delete  it after the  class is that ok?

**Review by shawn-ffufm — APPROVED** [review]

**Pull request merged**